CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 [Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-humio/compare/0.3.0...HEAD
 
+### Added
+
+-   Overriding of the Humio URL domain to support on-premise deployments of Humio.
```

Comment:
🤣 https://collectivecontent.co.uk/2018/04/19/should-i-say-on-premise-or-on-premises-it/